keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/SecurityGroup.kt

```diff
@@ -1,33 +1,22 @@
-/*
```

Comment:
Any reason you deleted this header?

Reply:
The file and class `SecurityGroup` was renamed to `SecurityGroupSpec` which still has the header. This is a brand new file. I'm under the impression we no longer add Netflix copyright headers to new files?

Reply:
Oh. 🤷‍♀ I thought we did! No idea.